File: Platform.FoodMe.Admin/Platform.FoodMe.Admin.ViewModels/ViewModels/AuthenticationData/RoleClaimVM.cs
using Platform.Shared.Permissions;

namespace Platform.FoodMe.Admin.ViewModels.ViewModels.AuthenticationData;

public class RoleClaimVM
{
    public Modules ClaimType { get; set; }
    public List<CRUDPermissions> ClaimValue { get; set; }
}

File: Platform.FoodMe.Admin/Platform.FoodMe.Admin/src/app/routes/ReferentialData/Models/User Data/RoleClaimVM.ts

import { CRUDPermissions } from "@shared/Models/Permissions/CRUDPermissions";
import { Modules } from "@shared/Models/Permissions/Modules";

export class RoleClaimVM{
  claimType : Modules = Modules.None;
  claimValue : Array<CRUDPermissions> = [];
}
